operations/app/src/modules/key_vault/main.tf

```diff
@@ -0,0 +1,71 @@
+terraform {
+  required_version = ">= 0.14"
+}
+
+locals {
+  object_ids = ["34232fe8-00ad-4bd0-9afb-eb9b3cc93ffe",
+                "cd341fbc-26a3-405c-a350-c4237a27aa93",
+                "637fb7df-c200-4e0d-ba86-608576acb786",
+                "aabc25d7-dd99-42b9-8f3a-fd593b1f229a"]
+}
+
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_key_vault" "application" {
```

Comment:
Could we add a [`lifecycle.prevent_destroy = true`](https://www.terraform.io/docs/language/meta-arguments/lifecycle.html) to this module? This contains data that would be damaging if accidentally ran a terraform destroy on it.

Reply:
Added